_tests/QITSelfTests.php

```diff
@@ -19,19 +19,19 @@ class Context {
 	public static $action;
 	public static $suite;
 	public static $test;
-	public static $sut_slug;
+
+	/*
+	 * To run tests in QIT, we need to assign the results to a plugin in the Marketplace.
+	 * We use the extension "woocommerce-product-feeds" and theme "storefront", because their owned by the test user in Staging.
```

Comment:
Minor typo:

```suggestion
	 * We use the extension "woocommerce-product-feeds" and theme "storefront", because they're owned by the test user in Staging.
```